hardhat.config.ts

```diff
@@ -69,7 +69,7 @@ const config: HardhatUserConfig = {
         url: process.env.POLYGON_NODE_URL || "",
         accounts: process.env.POLYGON_PRIVATE_KEY !== undefined ? [process.env.POLYGON_PRIVATE_KEY] : [],
         enabled: true,
-        blockNumber: 22877930     // Essential for mainnet forking !!
+        // blockNumber: 22877930     // Essential for mainnet forking !!
```

Comment:
```suggestion
        blockNumber: 22877930     // Essential for mainnet forking !!
```

Reply:
Alright unwanted comments removed and it's now forking from this block number